.github/workflows/nodejs-ci.yml

```diff
@@ -216,11 +221,29 @@ jobs:
         if: ${{ !cancelled() && !contains( inputs.excluded_jobs, 'build' )  }}
         run: |
           ${{ inputs.build_command }}
+      - name: Developer Tests
+        id: developer_tests
+        if: ${{ always() && inputs.run_dev_test && github.event.pull_request.base.ref == 'qa' }}
```

Comment:
Condition need to be set in such way that user is able to run test on any branch PR.

Reply:
udpated condition to take list of branches